RedHat-UX/red-hat-design-system
# docs(cta): light DOM CSS

## What I did

1. Attempted to replicate as much of CTA styles as light DOM CSS using the `:defined` pseudo-class as a demo.
2. Documented the light DOM CSS on the CTA Code docs page.


## Testing Instructions

1. Check the [CTA Code docs, Custom Light DOM CSS](https://deploy-preview-1183--red-hat-design-system.netlify.app/elements/call-to-action/code/#custom-light-dom-css).
2. Check the [proof-of-concept demo](https://deploy-preview-1183--red-hat-design-system.netlify.app/elements/call-to-action/demo/no-javascript-customization).
4. Check the [light DOM CSS file](https://github.com/RedHat-UX/red-hat-design-system/pull/1183/files#diff-b20fff004276871932356783cc8a2c619c817236ef490264484de5b4b3ad5bb1).

## Notes to Reviewers

I'm open to suggestions on how better to maintain and document this.
